src/Spectre.libGaussianMixtureModelling/DataTypes.h

```diff
@@ -16,6 +16,11 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
 #pragma once
-typedef double DataType;
+#include <random>
+#include "Common\DataTypes.h"
+
+namespace spectre::unsupervised::gmm
+{
+using RandomNumberGenerator = std::mt19937_64;
```

Comment:
I thought about it in `Common\DataTypes.h`. It will be useful also for `libGenetic`.

Reply:
Yeah I initially thought the same, but currently libGenetic and libClassifier aren't using those types just yet. If we happen to include them, we can certainly move this part.